bootstrap_cfn/config.py

```diff
@@ -51,7 +51,7 @@ class ConfigParser(object):
     def __init__(self, data, stack_name, environment=None, application=None):
         self.stack_name = stack_name
         self.data = data
-
+        self.stack_id = self.stack_name.split('-')[-1]
```

Comment:
how can you have access to the stack id here? since this generates the template to create the stack id to begin with.  You might be able to use  `AWS::StackId` (or maybe `AWS::StackName` ) with cloudformation syntax to obtain the ID.